core/src/test/kotlin/com/ritense/valtimo/camunda/authorization/CamundaBeanAuthorizationIT.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.valtimo.camunda.authorization
+
+import com.ritense.valtimo.BaseIntegrationTest
+import org.junit.jupiter.api.Test
+import java.util.UUID
+import javax.transaction.Transactional
+
+@Transactional
+class CamundaBeanAuthorizationIT: BaseIntegrationTest() {
+
+    @Test
+    fun `should be able to execute process bean without authorizing`() {
```

Comment:
This currently only tests the service task. The story describes some others. Shouldn't there be test-cases for those as well?

Reply:
I added test cases for all the examples named in the story. The expectation for the current solution is that it should also work for other applications that you can configure in bpmn.